commands/webui.go

```diff
@@ -54,6 +57,7 @@ Available git config:
 	flags := cmd.Flags()
 	flags.SortFlags = false
 
+	flags.StringVar(&options.host, "host", "127.0.0.1", "Network address or hostname to listen to (default 127.0.0.1)")
```

Comment:
This is nice. Could you have it in a separate PR?

Reply:
Sure. :)

Reply:
Here it is: https://github.com/MichaelMure/git-bug/pull/583